web/src/pages/map/doc.vue

```diff
@@ -175,9 +191,19 @@ export default VueParent.extend({
   z-index: 800;
 }
 
+#doc > div {
+  display: flex;
+  flex-direction: column;
+  min-height: 100%;
+}
+
 #doc_bottom {
   opacity: 0.5;
-  bottom: 0;
-  position: absolute;
+  margin-top: auto;
 }
+
+button {
+  margin-bottom: 10px;
+}
+
```

Comment:
Just use a standard bootstrap button.

Reply:
No. I does not see what you need to add CSS, just to show a new button.

Reply:
The margin-bottom is required because otherwise, the button is too close to `Source code` heading:

![image](https://user-images.githubusercontent.com/1451988/114403453-176f2080-9ba5-11eb-9566-b6ca2e6a6759.png)

I changed the rule to use [bootstrap `mb-3`](https://getbootstrap.com/docs/4.0/utilities/spacing/) instead.

Reply:
And for the other CSS modification, it fixes an issue where the doc footer would overlap with other content on smaller screen, eg previously:

![image](https://user-images.githubusercontent.com/1451988/114404630-2904f800-9ba6-11eb-8090-bf786ed6d950.png)

Now:

![image](https://user-images.githubusercontent.com/1451988/114404681-391cd780-9ba6-11eb-9cd8-f03c06fa635a.png)